paz/optimization/losses/frustumpointnet_loss.py

```diff
@@ -0,0 +1,243 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.losses import Huber
+from examples.frustum_pointnet.dataset_utils import NUM_HEADING_BIN, \
+    g_mean_size_arr, NUM_SIZE_CLUSTER
+
+def ExtractBox3DCornersHelper(centers, headings, sizes):
```

Comment:
this is a function it should start with a verb and use lower case separated by underscores. Also the world "Helper" does not  convey what it's doing. So we should try to be explicit